### ISSUE TITLE
Traps need to be created and added to the object placing system

### ISSUE THREAD

**Issue by KieranClarkee** [PlaceholderGames/2016-Team2Thirds#12]
The game needs a variety of traps that the hunter will be able to place in game so that they will have multiple options and choices to make which would be a benefit or a detriment.

**Comment by KieranClarkee** [comment]
Four different traps which include: RaisingTrap, FallingTrap, SlowingTrap and a RopeTrap have been created  and have been implemented into the player object placing system. I have done this buy adding to the current basic system where the player is able to place a wall and a floor piece. Instead of these two blueprints, I have added the four traps to the list and changed the inputs so that they would be created instead of the wall and the floor. The four traps had to be slightly altered in order to be able to work within the system.  These changes are below;

Instead of having the trap location being set by an EventBeginPlay, they are now set when the player actually walks over the trigger volume of the  trap. This makes sure that the trap would then spring in the correct location. Also I added a few parts to the traps to help resolve a few bugs. When the cage traps are triggered the player would be stunned for a period of time which would give the cage plenty of time to reach the new location without the player being able to just run straight out. The last change for the traps that I had to implement was that for the cage traps, the trigger would be destroyed on contact which would stop the cage from re-springing.

I consider this issue to be complete so I will now close it.
